client/src/components/RocketList/ListElements.js

```diff
@@ -27,6 +27,9 @@ export const RocketListContainer = Styled.div`
     align-items: flex-start;
     justify-content: flex-start;
     width: 100%;
+    @media (max-width: 540px) {
```

Comment:
and here you set the same media query. consider making this a function so that you can reuse it and change the media query size in one place. something like:

```
const smallBreakPoint = css => `
  @media (max-width: 540px) {
    ${css}
  }
`
```